src/pages/2021/sponsorlogistics.mdx

```diff
@@ -2,9 +2,105 @@ import { SiteLayout } from "../../components/layout/SiteLayout";
+
+**Attendee Types**
+
+- **Full-Conference Student Attendees -** People who are currently full-time or part time students.
+
+- **Full-Conference Professional Attendees -** These attendees may and are encouraged to attend any part of the conference. 
+
+- **Full-Conference Sponsor Attendees -** These attendees are attending the conference not as oSTEM members, but as oSTEM sponsors.
+
+- **Career Expo only Attendees -** These attendees do not need to register via the conference Bizzabo Portal - Only need to register to Brazen. They will receive an email invite from Brazen after the sponsor Booth Owner adds them as Booth Rep in Brazen.
+
+**Pre-Conference Oppertunities:**
+
+Sponsoring organizations are encouraged to participate in all parts of our conference, even those that happen prior to our event. Specifically, we encourage our sponsoring organizations to participate in the following parts of the program:
+
+- **Career and Graduate School Bootcamp - Thursday, October 19, 7:00PM-8:30PM ET -** Help conference attendees brush up on their resumes / CVs, elevator pitches, and interview skills prior to the expo. This is an opportunity to scout out attendees early, invite folks back to your booth at the expo, start to fill your interview roster, and make sure attendees are in top shape for the expo floor. Event registration link TBD.
+
+- **New to the Conference Session - TBD -** This session is especially helpful for those attendees who have never been to an oSTEM Conference before and want to learn the lay of the land. Conducted by members of the oSTEM Conference Committee, we’ll share ways to navigate the schedule and get the most out of your attendee experience. Event registration link TBD.
+
+**Programming Highlights**
+
+Sponsoring organizations are encouraged to participate in all parts of our conference, taking note of the participation limits set by the attendee type above. Specifically, we encourage our sponsoring organizations to participate in the following parts of the program:
+
+- **Career and Graduate School Expo -** Think pipe and drape booths, lines of eager attendees, and reviewing resumes / CVs to find your ideal candidates. Don’t be surprised if you see raffle drawings, on-site job/internship offers, iPads collecting contact information, and all the other trappings of your typical career fair. All sponsors with a booth will have the same booth for both Day 1 and Day 2 of the expo.
+
+    - Day 1 - Thursday, October 28, 3:00-7:00 ET (4 hours)
+
+    - Day 2 - Friday, October 29, 5:30-7:30 ET (2 hours)
+
+- **Poster and Hackathon Judging Session - Friday, November 13, 12-3:30 PM PST -** We need folks who are willing to learn more about the amazing research or hackathon project from this year's attendees. No prior experience is required. If you’re interested in being a poster or hackathon judge, please fill out the form linked at the button below.
+
+    - (button: Apply to be a Poster Judge)
```

Comment:
These links don't exist yet (@immortalbard , do they?), so let's just comment it out and say something like "Applications for poster and hackathon judges will open soon."

I think(?) you can do comments like 

```html
<--
stuff that should be commented goes here
-->
```

Reply:
https://form.jotform.com/212157738482158 <- link to poster judging submission

Reply:
Just added a link for the sponsored workshop button link: https://form.jotform.com/211587245359160

Reply:
The link is also added in the notion page.